classes/image_types_tegra.bbclass

```diff
@@ -32,6 +32,7 @@ TEGRA_BUPGEN_STRIP_IMG_NAMES ??= ""
 TEGRA_BUPGEN_STRIP_CMD ??= "${@tegraflash_bupgen_strip_cmd(d)}"
 
 DTBFILE ?= "${@os.path.basename(d.getVar('KERNEL_DEVICETREE').split()[0])}"
+INITRD_DTBFILE ?= "${@os.path.basename(d.getVar('INITRD_KERNEL_DEVICETREE').split()[0]) if d.getVar('INITRD_KERNEL_DEVICETREE') else '${DTBFILE}'}"
```

Comment:
I think it's OK to limit the `INITRD_KERNEL_DEVICETREE` variable to naming just one device tree, so the split is probably not needed here.  Also, it's probably better to default to a null string instead of ${DTBFILE} if it's not defined at all.

Reply:
Makes sense. This is resolved.